aws-transfer-profile/aws-transfer-profile.json

```diff
@@ -0,0 +1,128 @@
+{
+  "typeName": "AWS::Transfer::Profile",
+  "description": "Resource Type definition for AWS::Transfer::Profile",
+  "definitions": {
+    "Tag": {
+      "description": "Creates a key-value pair for a specific resource.",
+      "type": "object",
+      "properties": {
+        "Key": {
+          "type": "string",
+          "description": "The name assigned to the tag that you create.",
+          "minLength": 1,
+          "maxLength": 128
+        },
+        "Value": {
+          "type": "string",
+          "description": "Contains one or more values that you assigned to the key name you create.",
+          "minLength": 0,
+          "maxLength": 256
+        }
+      },
+      "required": [
+        "Key",
+        "Value"
+      ],
+      "additionalProperties": false
+    },
+    "CertificateId": {
+      "description": "A unique identifier for the certificate.",
+      "type": "string",
+      "pattern": "^cert-([0-9a-f]{17})$",
+      "minLength": 22,
+      "maxLength": 22
+    }
+  },
+  "properties": {
+    "As2Id": {
+      "description": "A unique identifier for the As2 account.",
+      "type": "string",
+      "minLength": 1,
+      "maxLength": 128
+    },
+    "ProfileType": {
+      "description": "Enum for type of profile.",
+      "type": "string",
+      "enum": [
+        "LOCAL",
+        "PARTNER"
+      ]
+    },
+    "Tags": {
+      "description": "An array of key-value pairs to apply to this resource.",
+      "type": "array",
+      "uniqueItems": true,
+      "maxItems": 50,
+      "insertionOrder": false,
+      "items": {
+        "$ref": "#/definitions/Tag"
+      }
+    },
+    "CertificateIds": {
+      "description": "An array of certificate id to apply to this resource.",
+      "type": "array",
+      "insertionOrder": false,
+      "items": {
+        "$ref": "#/definitions/CertificateId"
+      }
+    },
+    "Arn": {
+      "description": "Specifies the unique Amazon Resource Name (ARN) for the agreement.",
```

Comment:
`Specifies the unique Amazon Resource Name (ARN) for the profile.`

Reply:
++

Reply:
Good catch, updated!